### ISSUE TITLE
ClassyTaxiServer deployment failing under Windows

### ISSUE THREAD

**Issue by tfrysinger** [android/play-billing-samples#497]
I have followed the instructions for setting up the ClassyTaxiJava and ClassyTaxiServer samples. ClassyTaxiServer went fine (I was able to install the Firebase CLI under Windows, was able to perform 'NPM install', I was able to update the node packages so I didn't get any severe security errors reported, etc.), was able to do all the firebase use and firebase functions:config:set statements. Then I got to the 'firebase deploy' step, and I am seeing a variety of errors:

> firebase deploy

=== Deploying to 'staying-in-touch'...

i  deploying functions, hosting
Running command: npm --prefix "$RESOURCE_DIR" run lint

> functions@ lint C:\BLD\AndroidStudioProjects\ClassyTaxiServer
> tslint --project tsconfig.json

Running command: npm --prefix "$RESOURCE_DIR" run build

> functions@ build C:\BLD\AndroidStudioProjects\ClassyTaxiServer
> tslint -p tslint.json && ./node_modules/.bin/tsc

'.' is not recognized as an internal or external command,
operable program or batch file.

In my package.json file, I noticed the build script was defined like:

"build": "./node_modules/.bin/tslint -p tslint.json && ./node_modules/.bin/tsc"

I thought maybe the error pointed to the fact that it was trying to execute in that directory in a linux style "./" prefex and Windows didn't like that so I removed that prefix. This caused a different error:

> firebase deploy

=== Deploying to 'staying-in-touch'...

i  deploying functions, hosting
Running command: npm --prefix "$RESOURCE_DIR" run lint

> functions@ lint C:\BLD\AndroidStudioProjects\ClassyTaxiServer
> tslint --project tsconfig.json

Running command: npm --prefix "$RESOURCE_DIR" run build

> functions@ build C:\BLD\AndroidStudioProjects\ClassyTaxiServer
> tslint -p tslint.json && node_modules/.bin/tsc

'node_modules' is not recognized as an internal or external command,
operable program or batch file.

So I am not sure how to continue here...

**Comment by GiridharRai** [comment]
True. Facing the same error while deploying the server code from Windows. The same code works absolutely smoothly without changes on Mac.

**Comment by nkhar** [comment]
Hello @tfrysinger  @GiridharRai. As you pointed out this will work on Mac or Linux, because pathname with slash "/" is correct for those systems: ./node_modules/.bin/tslint. On Windows on the other hand paths uses reverse slash "\".

Another thing to note is on Mac and Linux systems: "." means current directory. On windows I don't think there is such a command.
Check this stack overflow post for further details:
https://stackoverflow.com/questions/48605192/is-not-recognized-as-an-internal-or-external-command-windows-10

try running: "node_modules\.bin\tslint"

Hope this helps.

Best regards,
Nikoloz
